src/update_agent/actor.rs

```diff
@@ -321,6 +348,36 @@ impl UpdateAgent {
         Box::pin(state_change)
     }
 
+    /// If there are interactive users logged in, return an error and postpone reboot.
+    fn check_interactive_sessions(
+        &mut self,
+        release: Release,
+    ) -> ResponseActFuture<Self, Result<Release, ()>> {
```

Comment:
I think it'd be much clearer if I could return a `ResponseActFuture<Self, bool>` here, so then later, in the calling function (`tick_finalize_update()`), we don't need to `match` on a `Result` type to determine whether we should postpone the reboot. Essentially, we're communicating a `bool` through `Ok()`/`Err()`. 
The Actix-related bits were a bit tricky for me so I wasn't able to figure out how to do this.